pkgs/top-level/python-packages.nix

```diff
@@ -3710,7 +3710,7 @@ in {
 
   libxslt = (toPythonModule (pkgs.libxslt.override {
     pythonSupport = true;
-    inherit python;
+    python3 = python;
```

Comment:
This looks awkward now, but this is what other packages do in this file.

Reply:
A more elegant way would be to remove this line and add disabled to the package. You need to input python and not python3 here because otherwise it would break the package for different python versions.

Reply:
> A more elegant way would be to remove this line and add disabled to the package.

Like `disabledIf (!isPy3k)`?

Reply:
What is here is fine.

Reply:
> disabledIf (!isPy3k)

That's for in case the bindings are no longer Python 2 compatible.

Reply:
> Like disabledIf (!isPy3k)?

in the package: ``disabled = !isPy3k;``.

Reply:
The package itself builds ok with python2 (it was the default on darwin after all), why disable it?

Reply:
I just answered your question. Packages schools not be null when a specific platform is not supported.